chore: doc review of self-hosted. (#146)
timescale/pgai: docs/self-hosting-vectorizer.md

# Run vectorizers in a self-hosted database using Vectorizer CLI

When you install pgai on Timescale Cloud or another cloud installation, you use 

> scheduling to control the times when vectorizers are run. A scheduled job checks in your 
> service checks if there is work to be done for the vectorizers. If there is, the job runs the cloud function to 

Reviewer: ```suggestion
scheduling to control the times when vectorizers are run. A scheduled job detects whether work is to be done for the vectorizers. If there is, the job runs the cloud function to 
```

       ```
     - The `VECTORIZER_DB_URL` environment variable:
       ```bash
       export VECTORIZER_DB_URL="postgres://user:password@host:port/dbname"
       vectorizer
       ```

      If you do not configure the connection string, the default value is 
      `postgres://postgres@localhost:5432/postgres`.

1. **Configure your API keys**

    For each vectorizer that you define in a self-hosted database, you must 
    [specify the name of the API key](./vectorizer-api-reference.md#embedding-configuration) used to 
    securely connect to the external embeddings provider. For a worker in Vectorizer CLI to 
    run the vectorizers in your database, you need to set the value of your API key in an environment 
    variable that matches the API key name defined for the vectorizer. For example, if your API key 
    name is `OPENAI_API_KEY`:
  
    ```bash
    export OPENAI_API_KEY="Your OpenAI API key"
    vectorizer
    ```

## Run vectorizers workers from Vectorizer CLI

By default, when you run a vectorizer worker, it loops over the vectorizers defined in 
your database and processes each vectorizer in turn. Five minutes after completing each 

> vectorizer run, the vectorizer worker loops over the vectorizers again. You can also the vectorizer
> `-i` / `--vectorizer-id` command line argument to manage which vectorizers that are run by that
> worker instance. You can use a vectorizer worker to:

Reviewer: ```suggestion
vectorizer run, the vectorizer worker loops over the vectorizers again. You can also use the 
`-i` / `--vectorizer-id` command line argument to manage which vectorizers that are run by that
worker instance. A vectorizer worker can:
```
